index.js

```diff
@@ -0,0 +1,154 @@
+function toggle(spanElement, arrowId) {
+  var arrow = document.getElementById(arrowId);
+
+  if (arrow.classList.contains("bi-caret-down-fill")) {
+    arrow.classList.remove("bi-caret-down-fill");
+    arrow.classList.add("bi-caret-up-fill");
+  } else {
+    arrow.classList.remove("bi-caret-up-fill");
+    arrow.classList.add("bi-caret-down-fill");
+  }
+}
+
+function filterFood() {
+  const priceRange = uislider.noUiSlider.get();
+  const minPrice = parseFloat(priceRange[0]);
+  const maxPrice = parseFloat(priceRange[1]);
+
+  const foodItems = document.getElementsByClassName("food-item");
+
+  for (let i = 0; i < foodItems.length; i++) {
```

Comment:
Maybe it's just a personal preference, but I think a forEach would look cleaner instead of for :D